### ISSUE TITLE
Remove text clipping on typeahead autosuggestions

### ISSUE THREAD

**Issue by cjg89** [UCF/Main-Site-Theme#354]
**Describe the bug**

https://github.com/UCF/Main-Site-Theme/blob/ba14233e19e3dd253e6584628a25049d3311defd/src/scss/components/_degree-search-typeahead.scss#L20-L28

https://github.com/UCF/Main-Site-Theme/blob/ba14233e19e3dd253e6584628a25049d3311defd/src/scss/components/_location-typeahead.scss#L20-L28

`overflow: hidden;`, `text-overflow: ellipsis;`, and `white-space: nowrap;` should be removed from the snippets linked above to ensure that users zoomed up to 200% will not have suggestion contents clipped that users at a standard zoom level would be able to see.

**Steps to Reproduce**

n/a

**Expected Behavior**

Text should never be clipped/truncated more when users zoom their browser up to 200%.

**Possible Fix**

Remove the rules noted above.

**Your environment:**

n/a

**Additional context**

https://www.w3.org/WAI/WCAG21/Understanding/resize-text
